Jenkinsfile

```diff
@@ -2,7 +2,7 @@
 
 String service = 'carto-renderer'
 boolean isPr = env.CHANGE_ID != null
-boolean lastStage
+String lastStage
```

Comment:
I noticed that I had put the wrong type when I saw this message:
"[tileserver » main #3](https://jenkins-build.socrata.com/job/tileserver/job/main/3/)
has failed in stage true"